packages/common/src/chains/goerli.json

```diff
@@ -67,6 +67,12 @@
       "block": 1561651,
       "consensus": "poa",
       "finality": null
+    },
+    {
+      "name": "berlin",
+      "block": null,
+      "consensus": "poa",
+      "finality": null
```

Comment:
Just some side note, in case anyone is wondering: these `consensus` and `finality` fields are from the old Casper/sharding days when it was still planned to do the consensus/finality switches on-chain, modeled this this way to be able to do the respective switches cleanly along the Casper HFs.

So these can very well removed, for the moment this is I guess rather distracting and confusing than anything else.

Reply:
@holgerd77 Should we remove those fields in this PR or in a separate one?

Reply:
Think I will merge here now since this is ready-for-merge here anyhow (thanks!). If you want you can open a separate PR on this, not too pressing though.